AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/DemoMenuActivity.kt

```diff
@@ -1,27 +1,28 @@
 package com.applovin.enterprise.apps.demoapp.ads
 
 import android.content.Context
-import android.content.Intent
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.AdapterView
 import android.widget.ArrayAdapter
+import android.widget.ListView
 import android.widget.TextView
 import com.applovin.enterprise.apps.demoapp.R
 import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem
-import kotlinx.android.synthetic.main.activity_list.*
 
 
 abstract class DemoMenuActivity : AppCompatActivity()
 {
+    protected lateinit var listView: ListView
```

Comment:
We usually have a new line separating field declarations
```suggestion
    protected lateinit var listView: ListView

```